heron/api/tests/java/org/apache/heron/streamlet/impl/StreamletImplTest.java

```diff
@@ -81,6 +82,22 @@ public void testBasicParams() throws Exception {
     assertEquals(bStreamlet.getChildren().size(), 0);
   }
 
+  @Test
+  public void testWithStream() {
```

Comment:
Test coverage can be useful for the following cases:
- if `streamId` is set duplicate (`test_stream2 == test_stream3`)
- if `streamId` is set `Bolt` level again as different from `Spout`'s `streamId`